### PR TITLE
Add GopherCon Brazil

### PR DESCRIPTION
**Pull request by ZeuCxb**
Please check if what you want to add to `awesome-go` list meets [quality standards](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#quality-standard) before sending pull request. Thanks!

**Please provide package links to:**
- github.com repo: 
- godoc.org: 
- goreportcard.com: 
- coverage service link (gocover, coveralls etc.): 

It`s a conference: https://gopherconbr.org

**Note**: that new categories can be added only when there are 3 packages or more.

**Make sure that you've checked the boxes below before you submit PR:**
- [x] I have added my package in alphabetical order
- [x] I know that this package was not listed before
- [x] I have added godoc link to the repo and to my pull request
- [x] I have added coverage service link to the repo and to my pull request
- [x] I have added goreportcard link to the repo and to my pull request
- [x] I have read [Contribution guidelines](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#contribution-guidelines) and [Quality standard](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#quality-standard).

Thanks for your PR, you're awesome! :+1:

### CODE REVIEW

**Comment by shurcooL** [comment]
LGTM, thank you.

File: README.md [hunk 1]
```diff
@@ -1297,6 +1297,7 @@ Where to discover new Go libraries.
 * [GolangUK](http://golanguk.com/) - London, UK
 * [GopherChina](http://gopherchina.org) - Shanghai, China
 * [GopherCon](http://www.gophercon.com/) - Denver, USA
+* [GopherCon Brazil](https://gopherconbr.org) - Florianópolis, BR
```
Comment (shurcooL) [inline README.md]: Although the spelling appears to be with an s rather than z on their site, are both okay?

**Comment by ZeuCxb** [comment]
In portuguese "Brasil" english "Brazil"

**Comment by shurcooL** [comment]
I understand, it's just that the conference site refers to itself as "GopherCon Brasil 2016" even on the english version:

https://2016.gopherconbr.org/en/

> GopherCon Brasil 2016 will take place on ...

image

But I guess it's not a big deal.

**Pull request merged**